don't build obsolete, unmaintained man pages, being phased out by
upstream (see PR 1291)

Index: doc/CMakeLists.txt
--- doc/CMakeLists.txt.orig
+++ doc/CMakeLists.txt
@@ -1,75 +1,3 @@
-
-### make and install the man pages
-if    (UNIX)
-	find_package(SevenZip)
-	set(AsciiDoc_FIND_QUIETLY TRUE)
-	find_package(AsciiDoc)
-	set(CREATE_MAN_PAGES)
-	if    (SEVENZIP_FOUND AND ASCIIDOC_FOUND AND XSLTPROC_FOUND AND DOCBOOK_FOUND)
-		execute_process(
-			COMMAND "${ASCIIDOC_BIN}"
-			RESULT_VARIABLE ASCIIDOC_RESULT
-			ERROR_QUIET
-			OUTPUT_QUIET
-			ERROR_STRIP_TRAILING_WHITESPACE)
-		if    (ASCIIDOC_RESULT EQUAL 0)
-			make_global_var(CREATE_MAN_PAGES TRUE)
-		else  (ASCIIDOC_RESULT EQUAL 0)
-			make_global_var(CREATE_MAN_PAGES FALSE)
-			message(STATUS "MAN pages will not be generated/installed (utility asciidoc is broken)")
-		endif (ASCIIDOC_RESULT EQUAL 0)
-	else  (SEVENZIP_FOUND AND ASCIIDOC_FOUND AND XSLTPROC_FOUND AND DOCBOOK_FOUND)
-		set(MISSING_UTILS "")
-		if    (NOT SEVENZIP_FOUND)
-			set(MISSING_UTILS "${MISSING_UTILS} 7z*")
-		endif (NOT SEVENZIP_FOUND)
-		if    (NOT ASCIIDOC_FOUND)
-			set(MISSING_UTILS "${MISSING_UTILS} asciidoc")
-		endif (NOT ASCIIDOC_FOUND)
-		if    (NOT XSLTPROC_FOUND)
-			set(MISSING_UTILS "${MISSING_UTILS} xsltproc")
-		endif (NOT XSLTPROC_FOUND)
-		if    (NOT DOCBOOK_FOUND)
-			set(MISSING_UTILS "${MISSING_UTILS} docbook-xsl")
-		endif (NOT DOCBOOK_FOUND)
-		message(STATUS "MAN pages will not be generated/installed; missing utilit(y|ies):${MISSING_UTILS}")
-	endif (SEVENZIP_FOUND AND ASCIIDOC_FOUND AND XSLTPROC_FOUND AND DOCBOOK_FOUND)
-
-	if    (CREATE_MAN_PAGES)
-		set(MAN_SOURCE_DIR "${CMAKE_CURRENT_SOURCE_DIR}/manpages")
-		set(MAN_BUILD_DIR  "${CMAKE_BINARY_DIR}/manpages")
-		set(MAN_DEST_DIR   "${MANDIR}/man6")
-
-		set(man_src
-				"${MAN_SOURCE_DIR}/spring.6.txt")
-		set(man_dst
-				"${MAN_BUILD_DIR}/spring-dedicated.6.gz"
-				"${MAN_BUILD_DIR}/spring-headless.6.gz"
-				"${MAN_BUILD_DIR}/spring-legacy.6.gz"
-				"${MAN_BUILD_DIR}/spring.6.gz")
-		set_source_files_properties(${man_dst} PROPERTIES GENERATED TRUE)
-
-		add_custom_command(
-			OUTPUT ${man_dst}
-			COMMAND "${MAN_SOURCE_DIR}/make_manpages.sh"
-				"${MAN_BUILD_DIR}"
-				"${ASCIIDOC_BIN}"
-				"${XSLTPROC_BIN}"
-				"${DOCBOOK_XSL}"
-				"${SEVENZIP_BIN}"
-			DEPENDS ${man_src}
-			WORKING_DIRECTORY "${MAN_SOURCE_DIR}"
-			COMMENT "manpages: Generating" VERBATIM
-		)
-
-		add_custom_target(manpages ALL DEPENDS ${man_dst})
-
-		install(FILES ${man_dst} DESTINATION "${MAN_DEST_DIR}")
-
-		message(STATUS "MAN pages will be installed to {INSTALL_PREFIX}/${MAN_DEST_DIR}")
-	endif (CREATE_MAN_PAGES)
-endif (UNIX)
-
 option(USERDOCS_PLAIN "Install plain/text engine user-docs (README, LICENSE, etc.)" FALSE)
 option(USERDOCS_HTML "Install HTML engine user-docs (README.html, LICENSE.html, etc.)" FALSE)
 
